Lib/argparse.py

```diff
@@ -1921,7 +1921,7 @@ def _parse_known_args(self, arg_strings, namespace):
         # find all option indices, and determine the arg_string_pattern
```

Comment:
This comment should be updated to reflect no longer looking only for indices.

Reply:
Done.